lib/enmeshed/connector_api.rb

```diff
@@ -0,0 +1,244 @@
+# frozen_string_literal: true
+
+module Enmeshed
+  class ConnectorError < StandardError; end
+
+  class ConnectorApi
```

Comment:
I have to admit that I only skimmed over this file and would need some guidance if a more detailed review is needed.

Reply:
During a call, we provided some guidance and made a few more suggestions. Therefore, I am resolving this thread as being completed.